addons/stock_account/views/stock_valuation_layer_views.xml

```diff
@@ -42,6 +42,7 @@
         <field name="model">stock.valuation.layer</field>
         <field name="arch" type="xml">
             <tree default_order="id desc" create="0"
+                  default_group_by="product_id"
```

Comment:
@Whenrow in Master by default the list is no longer grouped by product
this doesn't seem like it was untended added a groupby

Reply:
@HANNICHE-Walid it was intended ;)

Reply:
sorry I'll revert that